lib/MusicBrainz/Server/Constants.pm

```diff
@@ -36,7 +36,12 @@ our %EXPORT_TAGS = (
     editor             => _get(qr/^EDITOR_/),
     vote               => _get(qr/^VOTE_/),
     edit_status        => _get(qr/^STATUS_/),
-    access_scope       => _get(qr/^ACCESS_SCOPE_/),
```

Comment:
Unfortunately `_get` assumes that all variables are scalar, so moving `%ACCESS_SCOPE_BY_NAME` here required listing it explicitly.